src/common/runtime-stringify.js

```diff
@@ -0,0 +1,12 @@
+define([], function( ) {
+
+return function( args ) {
+	return JSON.stringify( args, function( key, value ) {
+		if ( typeof value === "function" ) {
+			return value.runtimeKey; // if undefined, the value will be filtered out.
```

Comment:
Not sure if we should do something about functions that don't have a runtimeKey.